<launch>
    <node pkg="f1tenth_verification" name="object_tracking_node" type="object_tracking_node.py" output="screen">
    </node>
</launch>
